### ISSUE TITLE
[Feature][Dashboard] All Dashboard features implementation follow-up

### ISSUE THREAD

**Issue by zzxwill** [oam-dev/kubevela#740]
Feature specification: https://docs.google.com/document/d/1jLOQRP6ffRJl2oo7QIALguKai5A3bH_eHpn9530r094/edit#heading=h.wlc81gyew8l

### 1. Application management
- [ ] create application (workloads/traits auto-rendering) #680 @majian159 @zzxwill 

### 2. Capabilities
- Workload
- [ ] list WorkloadDefinitions
- [ ] get details of WorkloadDefinition

- Operations
- [ ] list TraitDefinitions
- [ ] get TraitDefinition

**Comment by hongchaodeng** [comment]
@zzxwill 

I don't think we should rename traits to operations since it is based on OAM model. Let's keep using traits.

**Comment by zzxwill** [comment]
Agreed. @hongchaodeng  And @sunny0826 will follow your advice. Thanks.
